src/modules/sdk/objections/service.ts

```diff
@@ -1,11 +1,14 @@
 import { AxiosRequestConfig } from "axios";
 import logger from "../../../utils/logger";
 import { INTERNAL_API_URL } from "../../../utils/properties";
-import { getAxiosRequestConfig, HTTP_POST, makeAPICall } from "./axios.client";
+import { getAxiosRequestConfig, HTTP_PATCH, HTTP_POST, makeAPICall } from "./axios.client";
 import { ObjectionPatch } from "./types";
 
 const OBJECTIONS_API_URL = (companyNumber: string): string => `${INTERNAL_API_URL}/company/${companyNumber}/strike-off-objections`;
 
+const OBJECTIONS_API_PATCH_URL = (companyNumber: string, objectionId: string): string =>
```

Comment:
you should be able to replace the base part of the url 
```suggestion
const OBJECTIONS_API_PATCH_URL = (companyNumber: string, objectionId: string): string => OBJECTIONS_API_URL(companyNumber) + `/${objectionId}`;
```